tests/test_mgxs_library_condense/results_true.dat

```diff
@@ -1,49 +1,49 @@
-   material  group in nuclide      mean  std. dev.
-0         1         1   total  0.419289    0.01638   material  group in nuclide     mean  std. dev.
-0         1         1   total  0.07774   0.003273   material  group in  group out nuclide      mean  std. dev.
+   group in  material nuclide      mean  std. dev.
```

Comment:
I don't know that I have a preference as to the order of the columns, and it appears that the MGXS data here is the same, but I'm just curious.